src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -1213,6 +1213,7 @@ def load_arguments(self, _):
         c.argument('radius_server', min_api='2017-06-01', help='Radius server address to connect to.', arg_group='VPN Client')
         c.argument('radius_secret', min_api='2017-06-01', help='Radius secret to use for authentication.', arg_group='VPN Client')
         c.argument('client_protocol', min_api='2017-06-01', help='Protocols to use for connecting', nargs='+', arg_group='VPN Client', arg_type=get_enum_type(VpnClientProtocol))
+        c.argument('custom_routes', min_api='2019-02-01', help='Space-separated list of CIDR prefixes representing the custom routes address space specified by the customer for virtual network gateway and VpnClient.', nargs='+', arg_group='VPN Client')
```

Comment:
I have two questions about the role/function of custom_routes. What is the difference between address-prefixes and custom roues? It looks that custom routes not only work on VPN client, but also virtual network gateway. In this way, why do you want to place it in VPN Client Arguments group?

Reply:
In my understanding, there is a pair relationship between VPN client and virtual network gateway. You can set the `custom_routes` in the virtual network gateway, then VPN client can use it. If you check the commands under `az network vnet-gateway vpn-client`, there are just two commands `generate` and `show-url` without any CRUD commands.

Reply:
address-prefixes is used for P2S VPN client. Although they are using same structure, the content or usage are different.

Reply:
so. Based on the SDK's description, it's used by VPN client.